src/pages/OrganizationDetail.tsx

```diff
@@ -39,18 +46,31 @@ const OrganizationDetailPanelLoader = () => {
 	const params = useParams();
 	const { provider, organizationId = 'missing' } = params;
 	const [organization] = useOrganization(organizationId);
-
+	const [fields] = useBaseFields();
+	const [proposals] = useProposalsByOrganizationId(
+		PROPOSALS_DEFAULT_PAGE,
+		PROPOSALS_DEFAULT_COUNT,
+		organizationId,
+	);
+	const emptyProposalArray: FrontEndProposal[] = [];
+	const [proposalState, setProposalState] = useState(emptyProposalArray);
+	const emptyRecord: Record<string, string> = {};
+	const [fieldsState, setFieldsState] = useState(emptyRecord);
 	useEffect(() => {
```

Comment:
We could use a little semantic whitespace around these blocks. Let's put a new line before `useEffect`, to separate it from all the setup that precedes it.